osdep/timer-win2.c

```diff
@@ -19,22 +19,51 @@
 #include <sys/time.h>
 #include <mmsystem.h>
 #include <stdlib.h>
+#include <versionhelpers.h>
+
 #include "timer.h"
 
 #include "config.h"
 
 static LARGE_INTEGER perf_freq;
 
+// ms values
+static int hires_max = 50;
+static int hires_res = 1;
```

Comment:
> I'm tempted to default this to 4ms. Thoughts?

My main motivation to keep 1ms was to avoid regressions as much as possible (we don't actually know what might be affected, and in what ways, if we reduce the resolution).

But after some time, if it turns out that 4ms is good enough, I think it would be fine to change the default.

Reply:
Yep, that's reasonable